rest-new/src/main/java/org/jboss/pnc/rest/endpoints/ProductMilestoneEndpointImpl.java

```diff
@@ -132,12 +129,7 @@ public void cancelMilestoneClose(String id) {
 
     @Override
     public ValidationResponse validateVersion(VersionValidationRequest versionRequest) {
```

Comment:
add `@Valid` to VersionValidationRequest and I belive then you can rely on the fields not be null.